kernel/src/common/utils.rs

```diff
@@ -19,18 +19,17 @@
 #[macro_export]
 macro_rules! static_init {
     ($T:ty, $e:expr) => {
-        // Ideally we could use mem::size_of<$T>, uninitialized or zerod here
```

Comment:
The doc comment also contains a reference to the size issue which needs adjustment due to this change.